From 341e7939f6678f403ee551b28996dbae14cf6fe4 Mon Sep 17 00:00:00 2001
From: Joshua Strobl <joshua@streambits.io>
Date: Tue, 24 Nov 2020 01:10:21 +0200
Subject: [PATCH 1/1] Reduce exit idle time.

A logout followed immediately by a login would no longer produce working audio. Reduce the exit time from 20 seconds to 4 seconds to mitigate most cases.
---
 src/daemon/daemon-conf.c  | 2 +-
 src/daemon/daemon.conf.in | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/daemon/daemon-conf.c b/src/daemon/daemon-conf.c
index bcf7329..a26dfd4 100644
--- a/src/daemon/daemon-conf.c
+++ b/src/daemon/daemon-conf.c
@@ -70,7 +70,7 @@ static const pa_daemon_conf default_conf = {
     .disallow_exit = false,
     .flat_volumes = false,
     .rescue_streams = true,
-    .exit_idle_time = 20,
+    .exit_idle_time = 4,
     .scache_idle_time = 20,
     .script_commands = NULL,
     .dl_search_path = NULL,
diff --git a/src/daemon/daemon.conf.in b/src/daemon/daemon.conf.in
index 7409976..d7325b6 100644
--- a/src/daemon/daemon.conf.in
+++ b/src/daemon/daemon.conf.in
@@ -39,7 +39,7 @@ ifelse(@HAVE_DBUS@, 1, [dnl
 ; realtime-scheduling = yes
 ; realtime-priority = 5
 
-; exit-idle-time = 20
+; exit-idle-time = 4
 ; scache-idle-time = 20
 
 ; dl-search-path = (depends on architecture)
-- 
2.29.2

